policy/test_submount.te

```diff
@@ -0,0 +1,39 @@
+#################################
+#
+# Policy for testing NFS crosmnt
+#
+
+gen_require(`
+	attribute file_type;
+	type nfs_t;
+')
+
```

Comment:
The preferred approach for refpolicy-based policies is to use an interface rather than directly requiring the raw types and using raw allow rules, unless no interface exists that fits the need.  This is supposed to make policy less brittle and more modular.